pkg/install/install.go

```diff
@@ -228,7 +230,14 @@ func (i *Installer) startInstallation(ctx context.Context) error {
 	return err
 }
 
+// incrInstallPhase increments the install phase in Cosmos DB to allow installation to be resumed if it is interrupted
 func (i *Installer) incrInstallPhase(ctx context.Context) error {
+	if _, ok := i.env.(env.Dev); ok && !installPhaseBootstrapDoneOnce {
+		i.log.Print("development mode: bootstrap phase will be repeated once to ensure restartability")
+		installPhaseBootstrapDoneOnce = true
+		return nil
+	}
```

Comment:
This needs more thinking time, it's too raw to be merged.  Open an issue and try to get a discussion going.

Reply:
https://github.com/Azure/ARO-RP/issues/451